src/tools/ping/PingConnection.cpp

```diff
@@ -285,6 +287,13 @@ PingConnection::ProcessEvent(
             QuicConnection,
             (uint32_t)(ElapsedMicroseconds / 1000),
             (uint32_t)(ElapsedMicroseconds % 1000));
+
+        if (this->IsServer) {
+            if (!QUIC_SUCCEEDED(
+                    MsQuic->ConnectionSendResumptionTicket(QuicConnection, QUIC_SEND_RESUMPTION_FLAG_FINAL, 0, nullptr))) {
```

Comment:
Note: this succeeds even when 0-RTT the session parameter isn't set; investigate further why the default seems wrong.

Reply:
Can you also align the `MsQuic->Conn..` with the previous line and replaces `!QUIC_SUCCEEDED` with `QUIC_FAILED`?